lib/src/main/java/com/auth0/android/lock/Configuration.java

```diff
@@ -322,6 +327,14 @@ private void parseLocalOptions(Options options) {
         this.privacyURL = options.getPrivacyURL() == null ? "https://auth0.com/privacy" : options.getPrivacyURL();
     }
 
+    @StyleRes
+    public int authStyleForConnection(String strategy, String connection) {
+        if (authStyles.containsKey(connection)) {
+            return authStyles.get(connection);
+        }
+        return AuthConfig.styleForStrategy(strategy);
```

Comment:
about my comment `public static is hard to test since you can't mock/inject`
I was trying to note that with the static method there, the test for this method would have to repeat all the tests for the static method itself because you can't check that you're calling a static method